runtime/gc_structs/ClassIteratorClassSlots.cpp

```diff
@@ -32,29 +32,29 @@
 #include "ClassIteratorClassSlots.hpp"
 
 /**
- * @return the next slot in the class containing an object reference
+ * @return the next non-NULL object reference in a slot that contains one
```

Comment:
I think we can simplify this comment (and old one was not correct either referred to "object reference" but should be a class). What about just say `return the next non-NULL class reference`